src/types/types.ts

```diff
@@ -802,6 +824,26 @@ export interface EventSearchRequest extends Limit {
   search?: EventSearch;
 }
 
+/**
+ * Query schema for asset aggregate endpoint
+ */
+export interface EventAggregateQuery {
+  /**
+   * Filter on events with strict matching.
+   */
+  filter?: EventFilter;
+}
+
+/**
+ * Response from event aggregate endpoint
+ */
+export interface EventAggregateResponse {
```

Comment:
I am thinking maybe? because it's not a final response, it's just an item, what do you think?
```suggestion
export interface EventAggregate {
```

the same with assets